app/controllers/jita_controller.rb

```diff
@@ -0,0 +1,6 @@
+class JitaController < ApplicationController
+  layout 'items_application'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.